### PR TITLE
modal styling and functionality tweaks

### PR DESCRIPTION
**Pull request by 0xemc**
## Description

1. Added missing blur to Modal backdrop
2. Border radius issue and scroll bars fix for Windows
3.  Added escape key listener to close modal

## Changes

| Before  | After  |
|---------|--------|
|image|image|

-->

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

File: site/components/pages/Pledge/PledgeDashboard/index.tsx [hunk 1]
```diff
@@ -42,6 +42,18 @@ export const PledgeDashboard: NextPage<Props> = (props) => {
     setShowModal(false);
   };
 
+  /**
+   * Close the modal on escape press
+   * @todo extract as generic modal behaviour
+   */
+  useEffect(() => {
+    const escListener = (e: KeyboardEvent) =>
+      e.key === "Escape" && setShowModal(false);
```
Comment (ladytrekker) [inline site/components/pages/Pledge/PledgeDashboard/index.tsx:51]: I would check first if the modal is open 
`showModal && e.key === "Escape" && setShowModal(false);`

**Review by ladytrekker — COMMENTED** [review]
Thanks, haven't tested the PR. But the code looks straight forward.

See come comments below

File: site/locale/en-pseudo/messages.po [hunk 2]
```diff
@@ -6,6 +6,12 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: @lingui/cli\n"
 "Language: en-pseudo\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
```
[the remark this answers is not in the document]
Comment (0xemc) [reply site/locale/en-pseudo/messages.po:14]: Oh ok, I thought the line numbers would also need to be updated. No problem 👍

File: site/components/pages/Pledge/PledgeDashboard/index.tsx (hunk 1 above)
Comment (0xemc) [reply site/components/pages/Pledge/PledgeDashboard/index.tsx:51]: Sure thing

Comment (0xemc) [reply site/components/pages/Pledge/PledgeDashboard/index.tsx:51]: Sure thing

**Review by Atmosfearful — APPROVED** [review]

File: site/locale/en-pseudo/messages.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (0xemc) [reply site/locale/en-pseudo/messages.po:14]: Ah so it's just for the vercel preview build.. gotcha

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**